### DISCUSSION TITLE
Admin Garage/IED's

### DISCUSSION THREAD

**Discussion by benno1985** [Vdauphin/HeartsAndMinds#1008] [Question and answer]
I have gone through the parameters and set the Garage to 1.

class btc_p_garage { // Activate garage for admin:
        title = __EVAL(format ["      %1", localize "STR_BTC_HAM_PARAM_ARSENAL_GARAGEADMIN"]);
        values[]={0,1};
        texts[]={$STR_DISABLED,$STR_ENABLED};
        default = 1;

When local testing in editor, I cannot access the garage from the logistics box?

When I played a few years ago, you would walk up to a suspect item, interact to check for IED. Now its different?

**Comment by Vdauphin** [comment]
Hello,

> When local testing in editor, I cannot access the garage from the logistics box?

Sorry but this mission only work on dedicated server: http://vdauphin.github.io/HeartsAndMinds/#introduction-

> When I played a few years ago, you would walk up to a suspect item, interact to check for IED. Now its different?

Yes: http://vdauphin.github.io/HeartsAndMinds/InGame-documentation#ied

Have fun!

**Reply by benno1985** [reply]
I know the mission is only for a dedicated server. I Was wondering why I could not access the Garage playing from the editor to test?
We currently have another mission ongoing on our server, so I cannot test it at the moment.

**Reply by Vdauphin** [reply]
> I Was wondering why I could not access the Garage playing from the editor to test?

A quick look to the code of `btc_fnc_arsenal_garage` show it only works on dedicated server.

> We currently have another mission ongoing on our server, so I cannot test it at the moment.

You can start a dedicated server on your own computer to quickly test your change. I do it every time. Next to your client  arma3.exe you also have a arma3server.exe . Just get your server configuration from your "real" dedicated server (may be with some path change) and start it on your own computer.

**Reply by ghost** [reply]
If a vehicle is spawned via the admin garage, does it turn into a wreck and can be retrieved? Or does it simply get destroyed? 

Thanks!

**Reply by Vdauphin** [reply]
There are inside the H&M system: http://vdauphin.github.io/HeartsAndMinds/Add-vehicle-or-boat-to-H&M-system#2-in-game
